docs/execution-providers/MIGraphX-ExecutionProvider.md

```diff
@@ -24,18 +24,35 @@ For Nightly PyTorch builds please see [Pytorch home](https://pytorch.org/) and s
 
 Pre-built binaries of ONNX Runtime with MIGraphX EP are published for most language bindings. Please reference [Install ORT](../install).
 
-## Requirements
-
+As of ROCm version 6.0.2 AMD supplies pre-built debian binaries hosted on (https://repo.radeon.com/manylinux)
```

Comment:
Since ROCm version 6.0.2, AMD supplies pre-built debian binaries hosted on https://repo.radeon.com/rocm/manylinux.

Reply:
thanks will update

Reply:
Thanks for the update. 
How about the URL? https://repo.radeon.com/manylinux does not directly link to the binaries, https://repo.radeon.com/rocm/manylinux does.

Reply:
Updated